src/main/java/org/example/Main.java

```diff
@@ -4,7 +4,7 @@
 
 public class Main {
     public static void main(String[] args) {
-        List<String> gamesList = List.of("ボンバーマン","星のカービィ","スマッシュブラザーズ","ポケモン","ゼルダの伝説","ファイナルファンタジー","ロックマン","ドラゴンクエスト");
+        List<String> gamesList = List.of("ボンバーマン","星のカービィ","スマッシュブラザーズ","ポケモン","ゼルダの伝説","ファイナルファンタジー","ロックマン","ドラゴンクエスト","ペルソナ");
```

Comment:
カンマのあとに半角スペースをいれましょう。
IntelliJのactions on saveをONにすれば自動的にフォーマットしてくれるはずです。
```suggestion
        List<String> gamesList = List.of("ボンバーマン", "星のカービィ", "スマッシュブラザーズ", "ポケモン", "ゼルダの伝説", "ファイナルファンタジー", "ロックマン", "ドラゴンクエスト", "ペルソナ");
```

Reply:
修正したら、game/gameListブランチにてadd/commit/pushしましょう。
このPRのFiles changedに修正が反映されていることをセルフチェックした上で再度レビューを依頼ください。

Reply:
＞この PRのファイルが変更されましたに修正が反映されている事をセルフチェックした上で再度レビューを依頼してください。

gameListを編集いたしましたので、レビュー確認をお願い致します
![Uploading スクリーンショット 2023-09-14 13.32.42.png…]()